src/intervals/interval_operations/pointwise_boolean.jl

```diff
@@ -1,39 +1,27 @@
 # This file is part of the IntervalArithmetic.jl package; MIT licensed
 
-#=  This file contains functions not specified in the IEEE Std 1788-2015,
-    extending boolean operations defined for reals to intervals.
-
-    This is different from what is described in sections 9.5 of the
-    IEEE Std 1788-2015 (Boolean functions of intervals), since we deal here
-    with what is needed for drop-in replacement of floats by intervals,
-    and not what is reasonnable to do with intervals in a separate execution
-    environment.
-
-    Essentially, julia is better at composability and we are more ambitious
-    that what the standard ever expected, so we have some extra problems
-    to solve.
+#=  This file contains boolean operations defined for intervals.
 
     We use a trait system, defining the operation with an extra
-    `PointwisePolitic` argument defining how it should be handled.
+    `PointwisePolicy` argument defining how it should be handled.
 
-    The default operations use `IntervalArithmetic.pointwise_politic()`,
+    The default operations use `IntervalArithmetic.pointwise_policy()`,
     so in order to change the default behavior this function has to be
     redefined.
-
-    We define every operations using the default `PointwisePolitic{:ternary}()`
-    and then use it to define all the others.
 =#
 """
-    PointwisePolitic{P}
+    PointwisePolicy{P}
 
-Define which politic we use to extend pointwise comparison of
+Define which policy we use to extend pointwise comparison of
 
-Valid value for the politic identifier `P` are
+Valid value for the policy identifier `P` are
+    - `ieee1788` (default): Boolean operations as defined in the standard,
+      see section 10.5.10
```

Comment:
Maybe `:ieee1788` (which is too long) should be renamed to `:ieee`, or allow it as an alias.